From 2a81889c7d680fbe51de63dbbeb4e8c290ec46fa Mon Sep 17 00:00:00 2001
From: Matt Fleming <matt.fleming@intel.com>
Date: Fri, 21 Jun 2013 07:14:24 +0000
Subject: efi: Tag symbols with __export

I missed some symbols previously that are required to be exported when
loading vesamenu.c32.

Signed-off-by: Matt Fleming <matt.fleming@intel.com>
---
diff --git a/efi/main.c b/efi/main.c
index 51ff9f3..71333a4 100644
--- a/efi/main.c
+++ b/efi/main.c
@@ -21,7 +21,7 @@ uint32_t _IdleTimer = 0;
 char __lowmem_heap[32];
 uint32_t BIOS_timer_next;
 uint32_t timer_irq;
-uint8_t KbdMap[256];
+__export uint8_t KbdMap[256];
 char aux_seg[256];
 
 static inline EFI_STATUS
diff --git a/efi/vesa.c b/efi/vesa.c
index d259f60..b4a541b 100644
--- a/efi/vesa.c
+++ b/efi/vesa.c
@@ -297,7 +297,7 @@ static int efi_vesacon_font_query(uint8_t **font)
     return cp865_8x16_font_height;
 }
 
-int __vesacon_i915resolution(int x, int y)
+__export int __vesacon_i915resolution(int x, int y)
 {
 	/* We don't support this function */
 	return 1;
--
cgit v0.9.2
